tools/document-metrics/Cargo.toml

```diff
@@ -14,7 +14,7 @@ clap = { version = "4.1.4", features = ["derive"] }
 serde = { version = "1.0.114", features = ["derive"] }
 serde_json = "1.0.55"
 serde_yaml = "0.9.17"
-syn = "1.0.39"
+syn = { version = "1.0.39", features = ["full"] }
```

Comment:
Why this?

Reply:
This tool failed to compile, same locally, on the branch, it could not find `syn::Pat` enum, and according to the docs https://docs.rs/syn/latest/syn/enum.Pat.html this is available when feature `full` is enabled.

Reply:
The resolver bump introduces some [feature unification](https://doc.rust-lang.org/cargo/reference/resolver.html#features) that I haven't read, but may be caused by that?